aptos-move/framework/aptos-stdlib/sources/cryptography/multi_ed25519.move

```diff
@@ -191,47 +222,62 @@ module aptos_std::multi_ed25519 {
-            x"3040d9951e95f0a810b035453631045f6a3d887405bca808a305f3f2f729bf82044506ca122e9613191c2f378f4dac8868ddee3cd16c1c627c6e6f2abaf8660fac5697ef58896ed50ab4f1bbea62b2340743a685761a2569f3eecb80fe54432d6cb7f81a2e84ca9dce845b68970a26d8968ec72ec8bec29fda1a9559ebd06b019c5c562321b21a2c1a9c8352fd5748fb0985eca40736be075f973ed10b681a2fc17d4254ee3b313b87456c146d67c8316fd9f2df9902f3ea29a8b14bf1092607e0000000",
-            x"3040d9951e95f0a810b035453631045f6a3d887405bca808a305f3f2f729bf82044506ca122e9613191c2f378f4dac8868ddee3cd16c1c627c6e6f2abaf8660fac5697ef58896ed50ab4f1bbea62b2340743a685761a2569f3eecb80fe54432d6cb7f81a2e84ca9dce845b68970a26d8968ec72ec8bec29fda1a9559ebd06b019c5c562321b21a2c1a9c8352fd5748fb0985eca40736be075f973ed10b681a2fc17d4254ee3b313b87456c146d67c8316fd9f2df9902f3ea29a8b14bf1092607ad2076e059350e1d38813ddcefaa0f1056662105f6ea81a70341b4ebe36d8cf6ce5519e2c48ec8c81c462aac3e67345eda388e37b610bfd07147c9d4f187e6036c546e05978b496e46474893455fb624d4e00293f208a15762905d7bb7ebaf8de40c9615558179f5059fbb1f6bf41b25c533b1763fb9c80774028d0d1093640c9dbfdbf64a362914f350a45733d94baef60ddc3336a51750172df5c0533e196cb702a7c73a76f95424bd0ac610b228638283e497ef71967d99861530c1ffc80b06f145a86943f328bc9653cfe258fd3427d58700d6ce06d702787598a0b844fc472d9ab7bd07ae7d7ae80e645c1c67ec019992f718e8df11371ba7dfa1818603d24150a257e4c8b5d08b8611eb4335fb4db0d9f877e238e885afb70f477a2fe6975531d1a99c95c8b461ad03020bbb848f8b9b2bbb7546757e47e29f8fb0530c33840f2179501324ee3cc47519721a055e34069410626c10bc22f5ba080802724389c9dd89bc4ec77c3d24ab29c829a04106d14c432815adcac5fd56abccc50a0f5e8dea85f5c2e50052916f4ad263a00c2061cf4cfd4b0c08b3094ec3694056f1f22944c29e3c2bff41dd1df8d3af22ba2d06d1aa67c8895f2c62b84a3d7001c83811ad32be1f388480c67ae115072b19acd964bc7e5b236ce3a5c4581e7d58ef678a8486e6a9b25eeaaddf962d57d8ba03310f725fce96aba2943f64172d0e8f1d8133df1d55ae33eddbf6d92f8325887a6e859d850239e625a06dc73de808f47592ab7c02c20890bf72f88281ad204205aea85b60df984d044b644ef43c058a954195ad168c17a02468c97fa69908484994e3bd52e17df84ad4c38006de9f89310ed14606f8984a7c6fec6a4215ad14bc46e8759905c9031cea7c72e88906c0cc90cf713937e7bc53833116df149e99407d93a4854ed4f3e337310b9cb48d5041bdb75b256083d8329a3dce3aa4da1dc4d670e56898337e9078633fbe130b38e054ee5dc96dbe59f41d2e92cb72d9a4de44719596ea676d4ff50f7e62e78ee110a7347a4512ecde1134173aed804cdc79ddc7a41b410cd67d2e7095621800fffe0000",
-        ];
-
-        let num = vector::length(&ns);
-        let i = 0;
-        while (i < num) {
-            let pk_bytes = *vector::borrow(&pks, i);
-            let sig_bytes = *vector::borrow(&sigs, i);
-
-            let pk = new_validated_public_key_from_bytes(pk_bytes);
-            let pk = std::option::extract(&mut pk);
-            let pk = public_key_into_unvalidated(pk);
-            let sig = new_signature_from_bytes(sig_bytes);
-
-            assert!(signature_verify_strict(&sig, &pk, msg), 1);
-
-            i = i + 1;
-        }
+    fun test_gen_sign_verify() {
+        let (sk,pk) = generate_keys(5, 3);
```

Comment:
Test a few more thresholds here in a loop, as done in the previous version of the code:
```
let ks = vector[1, 1, 2, 2, 3, 15, ]; // the thresholds, implicitly encoded in the public keys
let ns = vector[1, 2, 2, 3, 10, 32, ];
```